src/app/designer/components/designer-node/designer-node.component.html

```diff
@@ -6,6 +6,10 @@
     <mat-icon color="primary">{{data.icon}}</mat-icon>
   </div>
 </div>
+<div class="actions">
+  <mat-icon color="warn"  (click)="removeNode()">delete</mat-icon>
+  <mat-icon color="primary" (click)="handleAction('refreshStep')">refresh</mat-icon>
```

Comment:
The designer is shared between editors so the only action that may be static is delete. All other actions must remain dynamic.